### PR TITLE
fix: resolve absolute path error in the alias options for css file

### LINKED ISSUE #1467: [Bug Report]: resolve absolute path error in the alias options
### Steps to reproduce

```
resolve： {
  alias: {
     '/@': path.resolve('src')
  }
}
```

### Reproduce link

_No response_

### What is actually happening?

resolve absolute alias error

### System Info

```Shell
mac
```

### PR DESCRIPTION
**Pull request by callqh**
**Description:**

**BREAKING CHANGE:**

**Related issue (if exists):**
close #1467

### CODE REVIEW

File: crates/plugin_css/src/dep_analyzer.rs [hunk 1]
```diff
@@ -70,7 +70,6 @@ impl Visit for DepAnalyzer {
 pub fn is_source_ignored(source: &str) -> bool {
   source.starts_with("http://")
     || source.starts_with("https://")
-    || source.starts_with('/')
```
Comment (wre232114) [inline crates/plugin_css/src/dep_analyzer.rs:73]: we cannot just remove this condition. for url(/logo.png), the resolver may throw 'can not resolve' error, but the image may exist under public dir, in this situation /logo.png should not be handled.

if think we should add more judgement in  resolve hook of plugin css, try alias resolve first but external unresolved /logo.png

**Review by wre232114 — CHANGES_REQUESTED** [review]

File: crates/plugin_css/src/dep_analyzer.rs (hunk 1 above)
Comment (callqh) [reply crates/plugin_css/src/dep_analyzer.rs:73]: Yes, I am trying to use the configuration in alias to determine whether further processing is needed.

File: crates/plugin_css/src/source_replacer.rs [hunk 2]
```diff
@@ -59,6 +62,8 @@ impl<'a> VisitMut for SourceReplacer<'a> {
                   } else {
                     self.public_path.trim_end_matches('/')
                   };
+                  println!("需要处理的路径{:?}==={:?}", source, normalized_public_path);
```
Comment (wre232114) [inline crates/plugin_css/src/source_replacer.rs:65]: remove println

File: crates/plugin_css/tests/analyze_deps.rs [hunk 3]
```diff
@@ -67,6 +77,7 @@ fn analyze_deps() {
     };
 
     css_plugin.analyze_deps(&mut params, &context).unwrap();
+    println!("{:?}", params.deps);
```
Comment (wre232114) [inline crates/plugin_css/tests/analyze_deps.rs:80]: remove println

File: crates/plugin_css/src/dep_analyzer.rs [hunk 4]
```diff
@@ -70,7 +76,19 @@ impl Visit for DepAnalyzer {
...
+    || source.starts_with("/")
     || source.starts_with("data:")
     || source.starts_with('#')
 }
+
+pub fn is_start_with_alias(alias: &HashMap<String, String>, source: &str) -> bool {
```
Comment (wre232114) [inline crates/plugin_css/src/dep_analyzer.rs:84]: alias may be a regex, refer to plugin resolve, i think a common alias matching util may need to be extracted to toolkit

**Review by wre232114 — CHANGES_REQUESTED** [review]

**Review by wre232114 — APPROVED** [review]

**Pull request merged**